code/game/objects/items/stacks/stack.dm

```diff
@@ -68,15 +67,6 @@
 				merge(S)
 				if(QDELETED(src))
 					return
-	var/list/temp_recipes = get_main_recipes()
-	recipes = temp_recipes.Copy()
-	if(material_type)
-		var/datum/material/M = SSmaterials.GetMaterialRef(material_type) //First/main material
-		for(var/i in M.categories)
-			switch(i)
-				if(MAT_CATEGORY_BASE_RECIPES)
```

Comment:
Can you preserve the category check somehow?

I know its rather useless with the way its structured now, but its important to statue & chiseling later when we add more category checks

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/62388554/4ce0d089-08d4-4454-80c6-fe7f6ce3433a)

Reply:
Though I guess I could fold it into `get_material_recipes`

Reply:
but well I will make it better for you